modules/ecs/cluster/logs.tf

```diff
@@ -0,0 +1,7 @@
+module "log_config" {
+  source              = "git@github.com:smartrent/terraform-infrastructure.git//modules/firelens_log_config?ref=5c3f0a306198a88137d07930586f5d1177bdaf46"
```

Comment:
Ah thinking on this a bit more, we should probably make this a private repo if we want to have refs to our internal repos. Or maybe set the log config in here directly instead of using a source that's internal. 🤔